.github/workflows/release.yaml

```diff
@@ -250,27 +246,23 @@ jobs:
       fail-fast: true
       matrix:
         config:
-        - target: x86_64-apple-darwin
-          runner: macos-13-large
-        - target: aarch64-apple-darwin
-          runner: macos-13-xlarge
-    env:
-      PYTHON_VERSION: "3.11"
-      MATURIN_VERSION: "1.4.0"
-      PROTOC: "${{ github.workspace }}/deps/protoc/bin/protoc"
+          - target: x86_64-apple-darwin
+            runner: macos-13-large
+          - target: aarch64-apple-darwin
+            runner: macos-13-xlarge
     steps:
       - uses: actions/checkout@v4
       - uses: extractions/setup-just@v1
         with:
           just-version: "1.23.0"
-      - name: install protoc
-        run: just protoc
+      - run: just protoc
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ env.PYTHON_VERSION }}
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
+          target: ${{ matrix.config.target }}
           args: --release ${{ matrix.config.target }} --out dist
           working-directory: bindings/python
           container: "off"
```

Comment:
here we pass release the target and out dist, also target is its own argument to the action.

Reply:
interestingly enough it was like that befoe, and so I made this swtich to see if it would fix things.

I'll revert back tho.

Reply:
sorry, we should, I think leave the `target:` key and _not_ pass anything to the `--release` flag, which doesn't appear to take any options.

Reply:
Yep sorry, to clarify:

- I see/understand the problem
- I (possibly wrongfully) believe I already tried doing that somewhere in this branch
- https://github.com/GlareDB/glaredb/commit/bb88f76e87486e09b17bc8a74a92a003181b2c1b introduced it which was a "fix" and so I got spooked

Regardless, I added a new commit to test this out: https://github.com/GlareDB/glaredb/pull/2564/commits/d54d21ee32264cea3150770be878116f1239462b

(fingers crossed).

If that commit fails, I'll make sure to comment out the other jobs (which I forgot to), so as to reduce noise and isolate the macos run

Reply:
I looked through the commits, and what you're suggesting is what I had intended in https://github.com/GlareDB/glaredb/pull/2564/commits/70221471cf2a6a023f398f0c829ea805926ab0c7 but forgot to delete the part after `--release`. My apologies!